plugins/labhub.py

```diff
@@ -104,3 +104,12 @@ def invite_cmd(self, msg, match):
             return self.INVITE_SUCCESS[team.lower()].format(invitee)
         else:
             return 'Nice try :poop:'
+
+    def callback_message(self, msg):
+        """
+        Invite the user whose message includes the holy "hello world"
+        """
+        if re.search(r'hello\s*,?\s*world', msg.body, flags=re.IGNORECASE):
+            user = msg.frm.nick
+            LabHub.invite(user, self.TEAMS['coala newcomers'])
```

Comment:
Nope, teams in github org are independent of one another, it doesn't lead to any bug.

However I should definitely check if the user is current member, would lead to reduction of reduntant invites when using hello world in a sentence :+1: